azurelinuxagent/ga/cgroupconfigurator.py

```diff
@@ -836,6 +819,12 @@ def start_extension_command(self, extension_name, command, cmd_name, timeout, sh
                         extension_name, ustr(exception))
                     self.disable(reason, DisableCgroups.ALL)
                     # fall-through and re-invoke the extension
+                except CGroupsException as exception:
```

Comment:
Adding this as a safeguard in case SystemdCgroupsApiv2 gets used to call start_extension_command

Reply:
Not needed, protected against enable flag